glitter_events/migrations/0005_migrate_location_locations.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from django.utils.text import slugify
+
+
+def shorten(text):
+    "Shorten text to 32 characters."
+    if len(text) > 32:
+        text = text[:28] + ' ...'
+    return text
+
+
+def revision(text):
+    "Try to set the revision off the text."
+    # We assume the revision is prepended like this '1) Some text'
+    revision = 0
+    if ')' in text:
+        revision, candidate = text.split(')', 1)
+        if revision.isdigit():
+            revision = int(revision)
+            text = candidate.strip()
+        else:
+            revision = 0
+
+    revision += 1
+    revision = str(revision) + ') '
+    return revision + text
+    
+
+def migrate(apps, schema_editor):
+    """
+    Forward migration
+    
+    The string field 'location' is moved to a many-to-many reference 
+    'locations' which therefore needs to be created. The title is shorter than
+    the location string in events, therefore we need to shorten it. Which also
+    means we need a mechanism to handle false deduplication.
+    """
+    Event = apps.get_model("glitter_events", "Event")
+    Location = apps.get_model("glitter_events", "Location")
+    dbi = schema_editor.connection.alias
```

Comment:
Don't think we need this?

Maybe drop this and the `.using(dbi)` from the statements?